pkg/drivers/none/none.go

```diff
@@ -61,24 +62,29 @@ type Config struct {
 
 // NewDriver returns a fully configured None driver
 func NewDriver(c Config) *Driver {
-	runner := command.NewExecRunner(true)
-	runtime, err := cruntime.New(cruntime.Config{Type: c.ContainerRuntime, Runner: runner})
-	// Libraries shouldn't panic, but there is no way for drivers to return error :(
-	if err != nil {
-		klog.Fatalf("unable to create container runtime: %v", err)
-	}
```

Comment:
There is just no point in initializing d.exec, d.runtime at this phase, since init is only called before the json marshal/unmarshal mechanism of libmachine, i.e. it would not survive that.